README.md

```diff
@@ -1 +1,132 @@
-# Paketo Composer Install Buildpack
+# PHP Composer Distribution Cloud Native Buildpack
+
+This buildpack runs the `composer install` command [composer](https://getcomposer.org/) to download project dependencies.
+It requires both `composer` and `php` on the path (see [requires](#requires)).
+
+A usage example can be found in the
+[`samples` repository under the `php/composer` directory](https://github.com/paketo-buildpacks/samples/tree/main/php/composer).
+
+## Detection
+
+Will add these requires/provisions to the build plan if and only if a `composer.json` file is found.
+
+### Requires:
+
+- `composer`
+- `php`
+
+### Provides:
+
+- `composer-packages`
+
+## Build
+
+Will run `composer install` in the project workspace to download project dependencies.
+The dependencies will be placed in a new layer and symlinked into the workspace at the 
+location specified by `COMPOSER_VENDOR_DIR`, which defaults to `vendor`.
+
+If dependencies are needed for Composer install scripts, use `BP_COMPOSER_INSTALL_GLOBAL`
+to specify which dependencies to install. 
```

Comment:
Let's mention this in the `Configuration` section so all of the configurations are together

Reply:
good catch. a lot of this README was written prior to the recent updates to https://github.com/paketo-buildpacks/rfcs/blob/main/text/php/0002-buildpack-yml-to-env-var.md